### PR TITLE
Add ward navigation feature with timeframe compatible

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**
Asana Link: https://app.asana.com/0/1204322126657135/1206514083860597/f

### CODE REVIEW

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 1]
```diff
@@ -128,6 +152,138 @@ export const CareViewPatients = () => {
     }
   };
 
+  const handleNext = () => {
+    if (currentShiftArray) {
+      const startHour = Number(
+        moment.unix(navHourEpoch.endHourEpoch).format("HH")
```
Comment (Arjun-Go) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:158]: I think we can use this DateTimeUtil function - epochTo24HourTimeFormat

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 2]
```diff
@@ -128,6 +152,138 @@ export const CareViewPatients = () => {
...
+  };
+
+  const handleNow = () => {
+    if (currentShiftArray) {
+      const currentEpoch = getPreviousNearbyHourEpoch(
+        Math.floor(new Date().getTime() / 1000)
+      );
+      const startTime = currentEpoch;
+      const endTime = getTimeInFuture(currentEpoch, timeframeLimitInHours);
+      const startHour = Number(moment.unix(startTime).format("HH"));
```
Comment (Arjun-Go) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:207]: Same util function here and the next line - epochTo24HourTimeFormat

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 3]
```diff
@@ -128,6 +152,138 @@ export const CareViewPatients = () => {
...
+    }
+  };
+
+  const handlePrevious = () => {
+    if (currentShiftArray) {
+      const startTime = getTimeInPast(
+        navHourEpoch.startHourEpoch,
+        timeframeLimitInHours
+      );
+      const startHour = Number(moment.unix(startTime).format("HH"));
```
Comment (Arjun-Go) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:252]: Here as well - epochTo24HourTimeFormat

File: src/features/CareViewPatientsSummary/components/CareViewPatientsSummary.jsx [hunk 4]
```diff
@@ -164,6 +177,33 @@ export const CareViewPatientsSummary = (props) => {
...
+              <>
+                <div className="treatments-notification">
+                  <WarningIcon />
+                  <span className="treatments-notification-span">
+                    {newTreatments + " New treatment(s): "}
+                    <>
+                      <Link
+                        data-testid="treatments-ipd-dashboard"
+                        onClick={() => {
+                          console.log("Inside treatments onclick function");
```
Comment (Arjun-Go) [inline src/features/CareViewPatientsSummary/components/CareViewPatientsSummary.jsx:190]: Do we need it here?

Comment (SooryaKumaranC-tw) [reply src/features/CareViewPatientsSummary/components/CareViewPatientsSummary.jsx:190]: No I will remove

File: src/features/CareViewPatients/components/CareViewPatients.jsx (hunk 1 above)
Comment (Arjun-Go) [reply src/features/CareViewPatients/components/CareViewPatients.jsx]: Oh, got it. Sure

**Review by Arjun-Go — APPROVED** [review]

**Pull request merged**